### DISCUSSION TITLE
Shows System.AccessViolationException

### DISCUSSION THREAD

**Discussion by webworn** [leap71/PicoGK#54] [Q&A] [answered]
Can anybody has this problem after running the code?
      0s    0.0+ Operating System      Microsoft Windows 10.0.22631
      0s    0.0+ Version:              Microsoft Windows NT 10.0.22631.0
      0s    0.0+ OS Architecture:      X64
      0s    0.0+ Proc Architecture:    X64
      0s    0.0+ 64 Bit OS:            Yes
      0s    0.0+ 64 Bit Process:       Yes
      0s    0.0+ Processor Count:      8
      0s    0.0+ Working Set:          21MB
      0s    0.0+ C# Framework:         .NET 8.0.8
      0s    0.0+ C# CLR Version:       8.0.8
      0s    0.0+ PicoGK Path:          picogk.1.7
      0s    0.0+ Command Line:         "C:\Users\MANISH\Documents\PicoGK Example\bin\Release\net8.0\PicoGK Example.dll"
      0s    0.0+ 
      0s    0.0+ ----------------------------------------
      0s    0.0+ 
      0s    0.0+ Welcome to PicoGK
Fatal error. System.AccessViolationException: Attempted to read or write protected memory. This is often an indication that other memory is corrupt.
Repeat 2 times:
--------------------------------
   at PicoGK.Library._Init(Single)
--------------------------------
   at PicoGK.Library.Go(Single, System.Threading.ThreadStart, System.String, System.String, System.String, System.String)
   at Program.<Main>$(System.String[])

**Comment by LinKayser** [comment]
Can you give us your system spec? Is this a regular Intel(x86)-based Windows machine?

**Reply by webworn** [reply]
Processor	Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz   1.80 GHz
Installed RAM	8.00 GB (7.88 GB usable)
System type	64-bit operating system, x64-based processor
AND 
Edition	Windows 11 Home Single Language
Version	23H2

**Reply by LinKayser** [reply]
That all looks normal to me. Does the older release work?

https://github.com/leap71/PicoGK/releases/tag/PicoGK-v1.7.0

Best,
Lin

**Reply by webworn** [reply]
0s    0.0+ Opened C:\Users\MANISH\PicoGK.log
      0s    0.1+ 
      0s    0.1+ ----------------------------------------
      0s    0.1+ 
      0s    0.0+ Current time (UTC): 2024-11-09 08:04:56 (UTC)
      0s    0.0+ Current local time: 2024-11-09 13:34:56 (+05:30)
      0s    0.0+ 
      0s    0.0+ ----------------------------------------
      0s    0.0+ 
      0s    0.0+ System Info:
      0s    0.0+ 
      0s    0.0+ Machine Name:         LAPTOP-NF7AMD6L
      0s    0.0+ Operating System      Microsoft Windows 10.0.22631
      0s    0.0+ Version:              Microsoft Windows NT 10.0.22631.0
      0s    0.0+ OS Architecture:      X64
      0s    0.0+ Proc Architecture:    X64
      0s    0.0+ 64 Bit OS:            Yes
      0s    0.0+ 64 Bit Process:       Yes
      0s    0.0+ Processor Count:      8
      0s    0.0+ Working Set:          21MB
      0s    0.0+ C# Framework:         .NET 8.0.8
      0s    0.0+ C# CLR Version:       8.0.8
      0s    0.0+ PicoGK Path:          picogk.1.7
      0s    0.0+ Command Line:         "D:\picogk\PicoGK Example\bin\Debug\net8.0\PicoGK Example.dll"
      0s    0.0+
      0s    0.0+ ----------------------------------------
      0s    0.0+
      0s    0.0+ Welcome to PicoGK
Fatal error. System.AccessViolationException: Attempted to read or write protected memory. This is often an indication that other memory is corrupt.
Repeat 2 times:
--------------------------------
   at PicoGK.Library._Init(Single)
--------------------------------
   at PicoGK.Library.Go(Single, System.Threading.ThreadStart, System.String, System.String, System.String, System.String)
   at Program.<Main>$(System.String[])
PS D:\picogk\PicoGK Example> 

Same

**Comment by webworn** [comment, marked as answer]
Problem resolved.

**Reply by LinKayser** [reply]
Can you tell us what it was?

**Reply by webworn** [reply]
dotnet sdk didn't installed properly via vs code due to some network issues.

**Reply by LinKayser** [reply]
Thank you!
